### DISCUSSION TITLE
ElectrumX Errors Since Install

### DISCUSSION THREAD

**Discussion by chadrbanks** [janoside/btc-rpc-explorer#268] [Q&A] [answered]
Umbrel v0.3.1
BTC RPC Explorer v2.0.2

When I click the app from Umbrel the homepage or dashboard seems fine, but when I try to go to certain pages I get the error below. A good example is when I search a wallet address, the transactions do not show up. Instead I get the follow error. I tried restarting Umbrel, this did not help. Any help would be appreciated.

Error: No ElectrumX connection available. This could mean that the connection was lost or that ElectrumX is processing transactions and therefore not accepting requests. This tool will try to reconnect. If you manage your own ElectrumX server you may want to check your ElectrumX logs.

JSON Format:
{
    "errorId": "23t07ug2wghefud",
    "error": {
        "error": "No ElectrumX Connection",
        "userText": "No ElectrumX connection available. This could mean that the connection was lost or that ElectrumX is processing transactions and therefore not accepting requests. This tool will try to reconnect. If you manage your own ElectrumX server you may want to check your ElectrumX logs."
    }
}

**Comment by janoside** [comment, marked as answer]
I haven't tested on Umbrel myself, but I believe that others were having ElectrumX-connection issues until they updated to BTC RPC Explorer v2.1.0.

Please try updating and see if the issue is resolved.

**Reply by chadrbanks** [reply]
Thank you for the info! I will have to see if that is something I can do or if Umbrel will need to push the update.

**Reply by chadrbanks** [reply]
Umbrel pushed the update, and that fixed it. Thanks again!

Umbrel v0.3.2
BTC RPC Explorer v2.1.0

**Reply by Minituff** [reply]
Hmm, I'm still having issues using these versions.

Umbrel v0.3.2
BTC RPC Explorer v2.1.0

**Reply by chadrbanks** [reply]
> Hmm, I'm still having issues using these versions.
> 
> Umbrel v0.3.2
> BTC RPC Explorer v2.1.0

Umbrel just pushed an update today, I want to say v0.3.3 or v0.3.4. You could try that.

**Comment by Jason213-wqwq** [comment]
Hallo, I have the same problem now! 
JSON Format:
{
"errorId": "23t07ug2wghefud",
"error": {
"error": "No ElectrumX Connection",
"userText": "No ElectrumX connection available. This could mean that the connection was lost or that ElectrumX is processing transactions and therefore not accepting requests. This tool will try to reconnect. If you manage your own ElectrumX server you may want to check your ElectrumX logs."
}
}

I tried to read one my address but the error is still there.
Hardware
Raspberry PI4 8Gb Ram
SSD disk 1 TB

Bitcoin Explorer version 3.1.1
Umbrel version 0.3.14
Just finished the sincronization (now 100%)

(why Electrum? I don't use Elecrum....)

How can I fix it?
Thank you
